systemtest/helpers_test.go

```diff
@@ -40,5 +40,7 @@ func withDataStreams(t *testing.T, f func(t *testing.T, unstartedServer *apmserv
 		srv := apmservertest.NewUnstartedServer(t)
 		srv.Config.DataStreams = &apmservertest.DataStreamsConfig{Enabled: true}
 		f(t, srv)
+		err = systemtest.Fleet.DeletePackage(systemtest.IntegrationPackage.Name, systemtest.IntegrationPackage.Version)
```

Comment:
This function will be removed in https://github.com/elastic/apm-server/pull/6568, so it's probably not worthwhile making this change. I don't think we can/should attempt to leave Elasticsearch in a pristine state after tests anyway, but happy to hear arguments.

Reply:
sounds good, seems like we're removing the data streams anyway which was why I added this. I will remove.